Contributing.md

```diff
@@ -39,6 +47,7 @@ Current Issue Triagers:
 Our most regular and experienced Issue Triagers sometimes move on to doing code reviews for pull requests, and have input into which pull requests should be merged. 
 
 Current Reviewers:
+- [@hwillson](https://github.com/hwillson)
 - [@lorensr](https://github.com/lorensr)
 - [@abernix](https://github.com/abernix)
```

Comment:
Thanks for adding me guys - I'm definitely looking forward to helping out more! (well, that is unless of course an Amazon S3 outage completely ruins my day and leads to me leaving tech completely ...)